Library/Formula/gdk-pixbuf.rb

```diff
@@ -52,12 +52,14 @@ def install
   def post_install
     # Change the version directory below with any future update
     ENV["GDK_PIXBUF_MODULEDIR"]="#{HOMEBREW_PREFIX}/lib/gdk-pixbuf-2.0/2.10.0/loaders"
+    ENV["GDK_PIXBUF_MODULE_FILE"]="#{lib}/gdk-pixbuf-2.0/2.10.0/loaders.cache"
```

Comment:
If it won't affect other formulae, that would be OK. But you still need to manually make symlink. (brew link happens before post_install). Also please use `make_symlink` instead of `ln`, so it will create symlink with related path.  

BTW, it would be great if you check other formulae depend on this one, especially those who regenerate cache in their own post install. Thanks.